### PR TITLE
Update Automatically pruning images module

### PR DESCRIPTION
**Pull request by bmcelvee**
@siamaksade - from the conversation in Slack and on the call

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review.

No QE review required.

File: modules/pruning-images.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ When the Pruning Custom Resource is deleted, the pruning `CronJob` and its relat
 
 .Procedure
 
-. Verify the Custom Resource Definition (CRD) contains the following `spec` and `status` fields:
+. Verify the object named `imagepruners.imageregistry.operator.openshift.io/cluster` contains the following `spec` and `status` fields:
```
Comment (mburke5678) [inline modules/pruning-images.adoc:23]: s/Verify the/ Verify that the

**Comment by mburke5678** [comment]
@bmcelvee One nit. LGTM

**Pull request merged**